src/Platform.Handlers/src/Xamarin.Platform.Handlers/Hosting/IStartup.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Xamarin.Platform.Hosting
+{
+	public interface IStartup
```

Comment:
I'm confused at the intention of this empty interface. What is this used for?

We usually don't create empty interfaces.

https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/interface
https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1040

In @PureWeen's prototype, there was a `Page` property:

https://github.com/PureWeen/app-contacts/commit/37b2a28939f7bf6ff9e0540d077e679dffbfb049#diff-a15127609366cfa5a2e48fa5414bfcd78d8fb16e46f53b4ba334eb8d82218594R8

```C#
    public interface IStartupPage
    {
        public Page Page { get; }
    }
```

This may be more preferable over the casting that is currently happening.